internal/v2/README.md

```diff
@@ -1 +1,56 @@
-This directory has implementation for S2Av2 go client.
+**This directory has implementation for S2Av2 go client**
+
+**How example certificates and keys generated in this directory:**
+
+**To create a self signed cert(and private key), Run the following commands using openssl:**
+
+openssl req -x509 -sha256 -days 7305 -newkey rsa:2048 -keyout root_key.pem -out
```

Comment:
Can we put all commands in triple ` blocks?

So it would look like:

```
openssl req -x509 -sha256 -days 7305 -newkey rsa:2048 -keyout root_key.pem -out root_cert.pem
```